pkg/runtime/runtime.go

```diff
@@ -46,7 +46,8 @@ func New(path string) (*Runtime, error) {
 		return nil, errs.Wrap(err, "Could not create runtime directory")
 	}
 
-	depot, err := newDepot()
+	runtimePath := path
+	depot, err := newDepot(runtimePath)
```

Comment:
Originally I wanted to pass `path` to `newDepot()`, but it didn't read right, so that's why I did the logic the way you originally saw it. I agree that the runtime shouldn't be concerned with what the depot can or cannot do, so I came up with something more readable here.